bld/namelist_files/namelist_defaults_pop.xml

```diff
@@ -1475,11 +1518,21 @@
 <riv_flux_shr_stream_file ocn_grid="gx1v6" estuary_type="'vsf_ebm'">ocn/pop/gx1v7/forcing/riv_nut.gnews_gnm.gx1v7_nn_open_ocean_nnsm_e1000r300_marginal_sea.20170425.nc</riv_flux_shr_stream_file>
 <riv_flux_shr_stream_file ocn_grid="gx1v7" estuary_type="'vsf_ebm'">ocn/pop/gx1v7/forcing/riv_nut.gnews_gnm.gx1v7_nn_open_ocean_nnsm_e1000r300_marginal_sea.20170425.nc</riv_flux_shr_stream_file>
 
+<riv_flux_shr_stream_file ocn_grid="gx1v7" estuary_type="'vsf_ebm'" rof_grid="rx1">ocn/pop/gx1v7/forcing/riv_nut.gnews_gnm.rx1_to_gx1v7_nn_open_ocean_nnsm_e1000r300_marginal_sea_170413.20190602.nc</riv_flux_shr_stream_file>
```

Comment:
I'm not sure why @matt-long's scripts that created these files included the extra initial date-like string. (It might actually be a PID.) But since those were the names of the generated files, I'd prefer to not change them, if we end up needing to later work out where they came from.

Reply:
The first date stamp is from the mapping file and the second is the date stamp on *this* file.